sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1133,7 +935,7 @@ def task_worker(self):
                 if lport not in self.port_dict:
                     continue
 
-                state = get_cmis_state_from_state_db(lport, self.xcvr_table_helper.get_status_sw_tbl(self.get_asic_id(lport)))
+                state = common.get_cmis_state_from_state_db(lport, self.xcvr_table_helper.get_status_sw_tbl(self.get_asic_id(lport)))
```

Comment:
Just a minor suggestion (though might not be directly related to this PR's topic of `refactor common xcvrd functions`):
would it better to encapsulate this long function call into a new method of CmisManagerTask, e.g. 
```
class CmisManagerTask(threading.Thread):
...
    def get_cmis_state_from_transceiver_status_table(self, lport):
        return get_cmis_state_from_state_db(lport, self.xcvr_table_helper.get_status_tbl(self.get_asic_id(lport)))
```
so that it would be simplified as: `self.get_cmis_state_from_transceiver_status_table(lport)`

Apart from xcvrd.py, I saw there are quite a few such long function calling `get_cmis_state_from_state_db(lport, self.xcvr_table_helper.get_status_sw_tbl(self.get_asic_id(lport)))` in test_xcvrd.py.